custom/icds/tests/test_ls_no_submissions.py

```diff
@@ -0,0 +1,126 @@
+        user = CommCareUser.create(cls.domain, name, 'password')
+        user.set_location(location)
+        UserESFake.save_doc(user._doc)
+        return user
+
+    @property
+    def today(self):
+        tz = pytz.timezone('Asia/Kolkata')
+        return datetime.now(tz=tz).date()
+
+    def test_form_sent_today(self, last_sub_time):
+        last_sub_time.return_value = {self.aww.get_id: self.today}
+        indicator = LSSubmissionPerformanceIndicator(self.domain, self.ls)
+        self.assertFalse(indicator.should_send())
+        self.assertEqual(indicator.get_messages(), [u'', u''])
+
+    def test_form_sent_seven_days_ago(self, last_sub_time):
+        last_sub_time.return_value = {self.aww.get_id: self.today - timedelta(days=7)}
+        indicator = LSSubmissionPerformanceIndicator(self.domain, self.ls)
+        self.assertFalse(indicator.should_send())
+        self.assertEqual(indicator.get_messages(), [u'', u''])
+
+    def test_form_sent_eight_days_ago(self, last_sub_time):
+        last_sub_time.return_value = {self.aww.get_id: self.today - timedelta(days=8)}
+        indicator = LSSubmissionPerformanceIndicator(self.domain, self.ls)
+        self.assertTrue(indicator.should_send())
+        messages = indicator.get_messages()
+        self.assertTrue('one week' in messages[0])
+        self.assertTrue('AWC' in messages[0])
+        self.assertEqual(messages[1], u'')
+
+    def test_form_sent_thirty_one_days_ago(self, last_sub_time):
+        # last submissions only looks 30 days into past
+        last_sub_time.return_value = {self.aww.get_id: None}
+        indicator = LSSubmissionPerformanceIndicator(self.domain, self.ls)
+        self.assertTrue(indicator.should_send())
+        messages = indicator.get_messages()
+        self.assertEqual(messages[0], u'')
+        self.assertTrue('one month' in messages[1])
+        self.assertTrue('AWC' in messages[1])
+
+    def test_nothing_from_last_sub(self, last_sub_time):
+        # last submissions only looks 30 days into past
+        last_sub_time.return_value = {}
+        indicator = LSSubmissionPerformanceIndicator(self.domain, self.ls)
+        self.assertTrue(indicator.should_send())
+        messages = indicator.get_messages()
+        self.assertEqual(messages[0], u'')
+        self.assertTrue('one month' in messages[1])
+        self.assertTrue('AWC' in messages[1])
+
+    def test_multiple_awc_eight_days_ago(self, last_sub_time):
+        aww_2 = self._make_user('aww_2', self.locs['AWC2'])
+        self.addCleanup(aww_2.delete)
+        last_sub_time.return_value = {
+            self.aww.get_id: self.today - timedelta(days=8),
+            aww_2.get_id: self.today - timedelta(days=8)
+        }
+        indicator = LSSubmissionPerformanceIndicator(self.domain, self.ls)
+        self.assertTrue(indicator.should_send())
+        messages = indicator.get_messages()
+        self.assertTrue('one week' in messages[0])
+        self.assertTrue('AWC, ' in messages[0])
```

Comment:
Changed it to awc1 https://github.com/dimagi/commcare-hq/pull/14993/commits/d14e67724a404267acf24e1120133e5808e2c7ee